docs/docs/guides/how-to-compose-your-own-testnet.md

```diff
@@ -0,0 +1,215 @@
+---
+title: How to build your own private, Ethereum testnet with eth-kurtosis
+sidebar_label: Build your own testnet from scratch
+slug: /how-to-compose-your-own-testnet
+toc_max_heading_level: 2
+---
+
+:::tip
+If you'd prefer to dive into the code example, visit the repository [here](https://github.com/kurtosis-tech/geth-lighthouse-package). 
+:::
+
+## Introduction 
+A testnet is an incredibly valuable tool for any web3 developer, no matter if you’re building a dApp or working on protocol-level changes. It comes as no surprise then that Ethereum has multiple public testnets in addition to a plethora of tools for local development networks (e.g. Ganache, Hardhat, Foundry Anvil).
+
+However, there are cases where an engineer  may need to develop or test functionality that: modifies the protocol itself (execution or consensus layers), necessitates a certain scale, or interacts with another blockchain entirely (e.g. L2s/rollups, bridges, or multi-chain relayers). In these cases, a fully functioning private testnet is required - one where the user has full control over every aspect of the network and its ancillary services. 
+
+:::note
+We will review the details on when and how a full, private testnet can be useful in another article.
+:::
+
+This guide will walk you through how to build your very own fully functioning, private Ethereum testnet using [`eth-kurtosis`](https://github.com/kurtosis-tech/eth-kurtosis/tree/main) components. In fact, the artifact you’ll end up with at the end of this tutorial will be a special type of environment definition that works at any scale you desire, is completely reproducible for CI workflows, and is modular - meaning you can add or remove other services to your network as you wish.
+
+**What you will do:**
+
+1. Create a local Kurtosis package template
+2. Within the template, define how your private Ethereum testnet should look like. This example will leverage the Lighthouse CL client and Geth EL client to build a single, full staking node over Docker with [`eth-kurtosis`](https://github.com/kurtosis-tech/eth-kurtosis/tree/main).
+3. Launch the private testnet locally over Docker
+4. Push everything to Github as a new repository for others to use
+
+**What you will need beforehand to get started:**
+
+- Install [Kurtosis](https://docs.kurtosis.com/install) (or [upgrade to latest](https://docs.kurtosis.com/upgrade) if you already have it)
+- Install [Docker & ensure its running](https://docs.kurtosis.com/install#i-install--start-docker)
```

Comment:
perhaps link out to docker before Kurtosis; as Kurtosis itself depends on Docker

Reply:
maybe we can point people to a GitPod as well